dev/com.ibm.websphere.appserver.features/visibility/public/xmlWS-3.0/io.openliberty.xmlWS-3.0.feature

```diff
@@ -0,0 +1,21 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.xmlWS-3.0
+visibility=public
+singleton=true
+IBM-App-ForceRestart: uninstall, \
+ install
+IBM-ShortName: xmlWS-3.0
```

Comment:
Please add: 
WLP-AlsoKnownAs: jaxws-3.0

Also add dependency on feature 
 com.ibm.websphere.appserver.eeCompatible-9.0 

And consider adding:

WLP-Activation-Type: parallel

Reply:
Also the NLS files may need to be updated to remove reference to JAX-WS

Reply:
Did you decide not to update the nls files for this feature?  They reference JAX-WS in them.